debian/control.in

```diff
@@ -4,7 +4,7 @@ Architecture: all
 Section: utils
 Priority: optional
 Depends: mono-runtime (>=4.0), ca-certificates-mono, libmono-microsoft-csharp4.0-cil, liblog4net1.2-cil, libnewtonsoft-json5.0-cil
-Maintainer: Michele Ippolito <m.ippolito@outlook.com>
-Description: Official KSP-CKAN client.
- Official client for the Comprehensive Kerbal Archive Network.
+Maintainer: The CKAN authors <authors@ksp-ckan.org>
```

Comment:
Make that <debian@ksp-ckan.org>, please. I am setting up the forwarding. Do you wish to receive those emails to Michele Ippolito <m.ippolito@outlook.com>?